### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by riccjohn**
This just adds a few lines on how to start the app and how to run the tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Apprenticeship project - tic-tac-toe written in TypeScript",
   "main": "index.js",
   "scripts": {
-    "test": "jest --watchAll"
+    "test": "jest --watchAll",
+    "start-dev": "webpack-dev-server --mode development --hot",
```
Comment (ssunday) [inline package.json]: I usually see just `start` for running the dev user

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.App {
+  margin: 1rem;
+}
+
+.container {
+  width: 100%;
+}
+
+.center-container {
```
Comment (ssunday) [inline src/App.css]: .scss/.css files I usually see as lower cased not upper cased

**Comment by ssunday** [comment]
This is documenting the app + linting it + adding styling...maybe break them out into separate PRs (next time you get a lot of work like this?)

File: src/App.css (hunk 2 above)
Comment (riccjohn) [reply src/App.css]: Yeah, I agree. For some reason I've seen react apps using capitalized filenames for .css stuff, and I'm not exactly sure why / when that's helpful. My guess is that it's supposed to be .css that's only pertinent to the corresponding component. If that's the case, I should move those css rules elsewhere.

Comment (rylnd) [reply src/App.css]: Yep, I've often seen this employed in a component-centric architecture. Thing.jsx, Thing.css, Thing.test.jsx, Thing.story.jsx, etc.

**Comment by rylnd** [comment]
@riccjohn because this branch is based on `reactjs` but you're requesting to merge in to `master`, you're implicitly including the entirety of the `reactjs` branch here as well. You have a few options:

1. if the work is truly independent, you could rebase this branch onto master (but I don't think it is)
2. You could change the base branch of this PR to be `reactjs`, so that it truly is only the few commits you were intending.
3. Ignore this PR until `reactjs` is merged.

You did a good job of separating concerns in your commits, so nice work there. I think that since this work is dependent on the react stuff, though, it doesn't make a lot of sense to review this in any other context. I personally would have kept this branch local and waited until after #14 was merged to open this one; it's just unnecessary noise to the reviewer until then.

**Pull request closed without merging**